actions/traits/LtiModuleTrait.php

```diff
@@ -47,7 +48,7 @@ protected function returnLtiError(\taoLti_models_classes_LtiException $error, $r
         } else {
             $launchData = \taoLti_models_classes_LtiLaunchData::fromRequest(\common_http_Request::currentRequest());
 
-            if ($launchData->hasReturnUrl()) {
+            if ($launchData->hasReturnUrl() && $error->getCode() != LtiErrorMessage::ERROR_UNAUTHORIZED) {
```

Comment:
The scenario I come up with where `$error->getCode() == LtiErrorMessage::ERROR_UNAUTHORIZED` is already taken to an error page. Do you have a scenario where this prevents the `launch_presentation_return_url` parameter from redirecting to it?